### DISCUSSION TITLE
为什么同步用户资料，不同步 lua 脚本

### DISCUSSION THREAD

**Discussion by yjnu** [rime/home#1669] [General]
两个问题：
1. 同步用户资料，不同步 lua 文件夹，需手动同步
2. 现在支持自定义图标，图标文件放在此目录中，也不同步

是我设置问题，还是程序就这样设计。虽不影响使用，但感觉程序逻辑还是有问题。我认为可以设计成同步目录下除 build 文件夹所有的文件与文件夹。

**Comment by Mc-GrowlR** [comment]
没准可以依赖第三方软件

**Comment by cvlot** [comment]
同步唯一的作用就是取出用户词库, 第三方同步软件很多

**Reply by yjnu** [reply]
哈哈，我用户词库关了，它这个同步功能更像是备份，备份配置文件到网盘目录好同步，不过 lua 脚本更改少，也习惯了。

**Reply by cvlot** [reply]
同步可以用fastcopy, 拷贝文件的神器, 速度更快, 用命令行可以一键同步或过滤你想要的目录, 小狼毫的同步是取出你输入置顶过的词, 说用户词库也不太严谨

**Reply by yjnu** [reply]
你说的是个好方法
